tests/vesting/FourYearVesting.js

```diff
@@ -1051,11 +1055,13 @@ contract("FourYearVesting", (accounts) => {
             // time travel
             await increaseTime(104 * WEEK);
 
+            await staking.setMaxVestingWithdrawIterations(maxWithdrawIterations);
+
             // withdraw
             tx = await vesting.withdrawTokens(root);
 
             // check event
-            expectEvent(tx, "TokensWithdrawn", {
+            expectEvent(tx, "TeamVestingCancelled", {
```

Comment:
this test looks like having an automatic search-and-replace issue  
it should not pass meaning there is highly likely an issue with vesting.withdrawTokens(...) contract function
pls also check other tests carefully

Reply:
Currently the event name for `_withdrawTokens` follow the same naming with the one in staking.
So, this is intended

Reply:
this is the wrong event for a regular token withdrawal. ot has nothing to do with a team vesting being cancelled.

Reply:
fixed in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/21dd284dc55c0b0355bb165ce2e24dbf154e32e6)